addon/mixins/components/tooltips.js

```diff
@@ -62,7 +63,7 @@ export default Ember.Mixin.create({
   destroyTooltip: on('willDestroyElement', function() {
     const tooltip = this.get('tooltip');
 
-    if (tooltip && Ember.typeOf(tooltip) === 'object') {
+    if (tooltip && tooltip instanceof Tooltip) {
```

Comment:
@sir-dunxalot  - take a look - I've updated this pull to do the `instanceof` check you suggested. It's worth noting that it looks like another attempt at fixing this bug was merged here https://github.com/sir-dunxalot/ember-tooltips/commit/432a943795c765f39a559dbe4421771eb41f9a99#diff-6d875f075dad177398e49011cc51f439R65 but that is simply not a good enough check; the collision in my case is w/ another object. (For others who might be reading this, https://github.com/Glavin001/ember-c3 uses a property `tooltips` - (a hash, therefore also an `object`) for it's graphs, so just by adding this tooltip library, any time you navigate away from a graph, your app crashes since this library is added to every component and hooks into the `willDestroyElement`). 

I've tested this w/ my app and it works for me, would love very much to get a release of this!